src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -20,13 +21,27 @@
  */
 public class ModelManager implements Model {
     private static final Logger logger = LogsCenter.getLogger(ModelManager.class);
-
     private final ContactList contactList;
     private final UserPrefs userPrefs;
     private final FilteredList<Person> filteredPersons;
+    private final UniqueModuleList moduleList;
+
+    /**
+     * Initializes a ModelManager with the given contactList, userPrefs and a moduleList.
+     */
+    public ModelManager(ReadOnlyContactList contactList, ReadOnlyUserPrefs userPrefs, UniqueModuleList moduleList) {
+        requireAllNonNull(contactList, userPrefs);
+
+        logger.fine("Initializing with contact list: " + contactList + " and user prefs " + userPrefs);
```

Comment:
The logger should include the input moduleList as well

Reply:
Right now the storage part hasn't been implemented, so it isnt initialized with anything right now.

Reply:
Remember to include the logging now!